tests/types/test_enum.py

```diff
@@ -0,0 +1,125 @@
+from enum import Enum
+
+import pytest
+from psycopg.types.enum import EnumInfo, register_enum
+
+from psycopg import pq
+
+
+class _TestEnum(str, Enum):
+    ONE = "ONE"
+    TWO = "TWO"
+    THREE = "THREE"
+
+
+@pytest.fixture(scope="session")
+def testenum(svcconn):
+    cur = svcconn.cursor()
+    cur.execute(
+        """
+        drop type if exists testenum cascade;
+
+        create type testenum as enum('ONE', 'TWO', 'THREE');
+        """
+    )
+    return EnumInfo.fetch(svcconn, "testenum")
+
+
+def test_fetch_info(conn, testenum):
```

Comment:
There is no test to fetch with an async connection. Could you please add one?

Reply:
Added general test, I guess it should be enough? Can you check it please?

Reply:
It looks very good, thank you